server/static/wrstat/src/Table.tsx

```diff
@@ -98,22 +100,28 @@ const noopFormatter = (a: { toString(): string }) => a + "",
 
 		return col["reverseFn"] ?? reverseSort(col["sortFn"] ?? noopSort);
 	},
-	makeRows = <T extends Record<string, any>>(rows: T[], cols: Column<T>[], onRowClick: (row: T) => void, rowExtra?: (row: T) => Record<string, any>) => {
+	makeRows = <T extends Record<string, any>>(rows: T[], cols: Column<T>[], onRowClick?: (row: T) => void, onRowHover?: (row: T) => void, onMouseOut?: (row: T) => void, rowExtra?: (row: T) => Record<string, any>) => {
 		return rows.map((row, n) => <tr
 			key={`table_row_${n}`}
 			role="button"
 			onKeyPress={e => {
 				if (e.key === "Enter") {
-					onRowClick(row);
+					onRowClick?.(row) ?? {};
```

Comment:
The nullish coalescing isn't needed here; I'm assumming that this syntax was copied from below where the result is being spread as extra params, so it needs the expression to result in an object.

That isn't required here, the optional chaining will suffice.